showcase.json

```diff
@@ -1,6 +1,21 @@
-[
+[{
+    "id": "motokolearn",
+    "name": "MOTOKOLEARN",
+    "oneLiner": "Motoko package meant to facilitate on-chain training and inference of machine learning models where having a large GPU is not a requirement",
+    "website": "https://github.com/ildefons/motokolearn",
+    "tags": [
+      "Tools / Infrastructure"
+    ],
+    "twitter": "https://twitter.com/ildefons",
```

Comment:
We generally request the company or project Twitter. Can you please update or remove?

Reply:
I am sorry I am not a company. I am dev that created an open source project. So I will remove this part